release-notes.txt

```diff
@@ -1,3 +1,228 @@
+
+#4527: gnrc_ipv6: Multicast is not forwarded if routing node listens to the
+       address
+#5016: gnrc_rpl: Rejoining RPL instance as root after reboot messes up rooting
+#5051: Forwarding a packet back to its link layer source should not be allowed
+#5230: gnrc_ipv6: multicast packes are not dispatched to the upper layers
+#5748: gnrc: nodes crashing with too small packet buffer
+#6123: gnrc: crash with (excessive) traffic in native
+#7474: 6lo: gnrc fragmentation expects driver to block on TX
+#7737: pkg: libcoap is partially broken and outdated
+#8086: gnrc_rpl_p2p: not compiling
+#8130: gcoap: can't build witth network stacks other than gnrc
+#8172: gnrc_netif, gnrc_uhcpc: Replacing prefix on border router results in no
+       configured prefix
+#8271: app/netdev: application stops working after receiving frames with
+       assertion or completely without error
+#9616: at86rf2xx: can not correctly set channel after `NETOPT_STATE_RESET`
+#9656: gnrc/netif: various problems after resetting interface a second time
+#9573: pkg/lwip: assert on samr21-xpro
+#9584: drivers/cc2420: add netopt NETOPT_MAX_PACKET_SIZE
+
+Timer related issues
+--------------------
+
+#5103: xtimer: weird behavior of tests/xtimer_drift
+#5338: xtimer: xtimer_now() is not ISR safe
+#6052: tests: xtimer_drift_gets_stuck
+#7114: xtimer: adds items to the wrong list if the timer overflows between
+       _xtimer_now() and irq_disable()
+#7347: xtimer_usleep stuck for small values
+#8251: telosb: timer config wrong when running o a tmote sky
+#9052: misc issues with tests/trickle
+
+Native related issues
+---------------------
+
+#495: native not float safe
+#5796 native: tlsf: early malloc will lead to a crash
+#6442: cpu/native: timer interrupt issue
+#7206: native: race-condition in IPC
+
+Other platforms related issues
+------------------------------
+
+#4512: pkg: tests: RELIC unittests fail on iotlab-m3
+#4612: pkg: TLSF does not build for 16 bit platforms
+#4872: at86rf2xx: Simultaneous use of different transceiver types is not
+       supported
+#4954: chronos: compiling with -O0 breaks
+#4866: periph: GPIO drivers are not thread tafe
+#5009: RIOT is saw-toothing in energy consumption (even when idling)
+#5128: some use of asm keyword might be missing volatile
+#5486: at86rf2xx: lost interrupts
+#5774: cpu: cortexm_common: context switching code breaks when compiling with
+       LTO
+#5848: arduino: Race condition in sys/arduino/Makefile.include
+#6018: nRF52: gnrc 6lowpan ble memory leak
+#6437: periph/spi: Leftovers from SPI rework in #4780
+#6567: periph/spi: Switching between CPOL=0,1 problems on Kinetis with software
+       CS
+#6650: GPIO Port shift in cc2538 is unconsistent
+#6838: Issuing a reboot on ATmega328p makes the board loop-reboot
+#6874: SAMD21: possible CMSIS bug
+#7020: isr_rfcoreerrors while pinging between CC2538DKs
+#7667: sam0 flashpage_write issues
+#7846: stm32f1: I2C read functions return bogus values with unconnected devices
```

Comment:
This was fixed/closed by the I2C refactoring

Reply:
done!